bin/build-ubuntu.sh

```diff
@@ -54,6 +61,15 @@ fi
 if [ ! -d "${DIR_THIRDPARTY}/ntf-core" ]; then
     git clone https://github.com/bloomberg/ntf-core.git "${DIR_THIRDPARTY}/ntf-core"
 fi
+# prometheus-cpp and its dependency for the plugin
+if [ "${BUILD_PLUGINS}" == true ]; then
+    if [ ! -d "${DIR_THIRDPARTY}/curl" ]; then
+        git clone https://github.com/curl/curl.git "${DIR_THIRDPARTY}/curl"
```

Comment:
We link all dependencies statically to our enterprise plugins, so we decided to link libcurl to the Prometheus plugin statically as well. There is a static version of the library, distributed via apt manager as a part of 'libcurl4-openssl-dev' package. Unfortunately, it has been built without support of position-independent code, thus, it can't be linked with shared object (like our plugin). Moreover, the distributed version depends on shared third party libraries, responsible for various features, which we don't even use in our plugin. So, in order to decrease the number of dependencies and to keep the tradition of linking everything statically, I've built a static version of libcurl with position-independent code enabled and without any additional dependencies.

Reply:
Makes sense!

Reply:
d